examples/jwt-auth/src/jwt.rs

```diff
@@ -480,10 +476,7 @@ mod tests {
 
         // Let's create a request with a badly formatted authorization header
         let request_with_too_many_spaces_in_auth = RouterRequest::fake_builder()
-            .headers(vec![(
-                HeaderName::from_static("authorization"),
-                HeaderValue::from_static("Bearer  "),
-            )])
+            .header("authorization", "Bearer")
```

Comment:
the `Bearer` value is now missing the extra spaces from the original test

Reply:
Nice ctach.
[f284020](https://github.com/apollographql/router/pull/908/commits/f2840200468686cddcad9b6dc60f39de9f33b61a)